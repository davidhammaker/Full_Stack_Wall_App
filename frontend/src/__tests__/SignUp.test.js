import React from 'react';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';
import SignUp from '../components/signup';


configure({ adapter: new Adapter() });


it('displays page correctly', () => {
  const signUp = renderer.create(
    <SignUp />
  );
  let tree = signUp.toJSON();
  expect(tree).toMatchSnapshot();
});
