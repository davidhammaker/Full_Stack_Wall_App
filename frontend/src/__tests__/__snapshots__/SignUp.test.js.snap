// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays page correctly 1`] = `
<div>
  <div
    className="btn-group float-right bg-dark mr-2 rounded sm"
  >
    <a
      className="text-light text-decoration-none"
      href="/"
    >
      <button
        className="btn btn-dark"
      >
        Home
      </button>
    </a>
  </div>
  <h1
    className="p-4 border border-left-0 border-top-0 border-right-0"
  >
    Sign Up
  </h1>
  <form
    className="p-4"
    onSubmit={[Function]}
  >
    <small />
    <div
      className="form-group"
    >
      <label>
        Username:
      </label>
      <input
        className="form-control"
        onChange={[Function]}
        type="text"
      />
    </div>
    <div
      className="form-group"
    >
      <label>
        Email:
      </label>
      <input
        className="form-control"
        onChange={[Function]}
        type="email"
      />
    </div>
    <div
      className="form-group"
    >
      <label>
        Password:
      </label>
      <input
        className="form-control"
        onChange={[Function]}
        type="password"
      />
    </div>
    <div
      className="form-group"
    >
      <label>
        Confirm Password:
      </label>
      <input
        className="form-control"
        onChange={[Function]}
        type="password"
      />
    </div>
    <div
      className="d-flex justify-content-end"
    >
      <input
        className="btn btn-info mt-2"
        type="submit"
        value="Submit"
      />
    </div>
  </form>
</div>
`;
